src/core/src/package_managers/ZypperPackageManager.py

```diff
@@ -159,11 +162,11 @@ def invoke_package_manager(self, command):
                 self.log_success_on_invoke(code, out)
 
             if code == self.zypper_exitcode_zypper_updated or code == self.zypper_exitcode_reboot_required:
```

Comment:
Remove if statement. __handle_zypper_updated_or_reboot_exit_codes() will be called always irrespective of error code